packages/runner/src/app/app.jsx

```diff
@@ -121,6 +196,13 @@ class App extends Component {
     })
   }
 
+  _onSpecListResizeEnd = () => {
+    this.isSpecListResizing = false
+    this.props.eventManager.saveState({
+      specListWidth: this.props.state.specListWidth,
+    })
```

Comment:
Need to add `specListWidth` to the allowed keys in [server/lib/saved_state.js](https://github.com/cypress-io/cypress/blob/f35efca744f5f57f061d9d00f50c3cf76474a57d/packages/server/lib/saved_state.js#L14).

Reply:
Done, thanks.

I haven't added any tests in the `runner` package for this (eg, click a spec, verify it executed) because I expect more changes as part of the unification project. I'll definitely add a lot more test around this once we are nearing a public release.